diagnostic.md

```diff
@@ -35,21 +35,16 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+ absolute, it shows the user location
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+ relative, it already exists we don't need to duplicate
```

Comment:
It would actually be an absolute path. In order to find the font, we need the root path (URL) to access the correct photo.